packages/react/src/__tests__/ReactProfiler-test.internal.js

```diff
@@ -1110,13 +1178,14 @@ describe(`onRender`, () => {
 
           Scheduler.unstable_advanceTime(5); // 0 -> 5
 
-          ReactTestRenderer.create(
+          ReactDOM.render(
```

Comment:
The assertion below fails when using createRoot(). 

`expect(mountCall[4]).toBe(5)` get the value of `mountCall[5]` - 22/19

I'm not sure if this is an expected change. Any ideas?